frontend/yarn.lock

```diff
@@ -1098,13 +1098,20 @@
   dependencies:
```

Comment:
Same as above, this file should be removed from the repository.

Reply:
`yarn.lock` is not a temporary lock-file and should therefore be kept in VCS; https://classic.yarnpkg.com/en/docs/migrating-from-npm